frontend/src/components/Forms/incidents/AddIncidentDetailForm.js

```diff
@@ -0,0 +1,149 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import CustomPropTypes from "@/customPropTypes";
+import { Field, reduxForm } from "redux-form";
+import { Form, Col, Row } from "antd";
+import * as FORM from "@/constants/forms";
+import { renderConfig } from "@/components/common/config";
+
+import { required, maxLength, number, dateNotInFuture } from "@/utils/Validate";
+
+const propTypes = {
+  incidentDeterminationOptions: CustomPropTypes.options.isRequired,
+  doSubparagraphOptions: CustomPropTypes.options.isRequired,
+  inspectors: CustomPropTypes.options.isRequired,
+  incidentStatusCodeOptions: CustomPropTypes.options.isRequired,
+  doDetermination: PropTypes.string,
+};
+
+const defaultProps = {
+  doDetermination: "PEN",
+};
+
+class AddIncidentDetailForm extends Component {
+  validateDoSubparagraphs = (value) =>
+    value.length === 0 ? "This is a required field" : undefined;
+
+  render() {
+    return (
+      <Form layout="vertical">
+        <Row gutter={48}>
+          <Col>
+            <h4>Incident Details</h4>
+            <Form.Item>
+              <Field
+                id="incident_timestamp"
+                name="incident_timestamp"
+                label="Incident Date and Time*"
+                placeholder="Please select date and time"
+                component={renderConfig.DATE}
+                showTime
+                validate={[required, dateNotInFuture]}
+              />
+            </Form.Item>
+            <Form.Item>
+              <Field
+                id="number_of_fatalities"
+                name="number_of_fatalities"
+                label="Number of Fatalities:"
+                component={renderConfig.FIELD}
+                validate={[number, maxLength(10)]}
+              />
+            </Form.Item>
+            <Form.Item>
+              <Field
+                id="number_of_injuries"
+                name="number_of_injuries"
+                label="Number of Injuries:"
+                component={renderConfig.FIELD}
+                validate={[number, maxLength(10)]}
+              />
+            </Form.Item>
+            <Form.Item>
+              <Field
+                id="emergency_services_called"
+                name="emergency_services_called"
+                label="Were emergency services called?*"
+                placeholder="Please choose one"
+                component={renderConfig.RADIO}
+                data={[{ id: "yes", label: "Yes" }, { id: "no", label: "No" }]}
```

Comment:
radio, True false, this seems redundant.

Reply:
Removed, don't need data.